library/include/flipper/posix/pio.h

```diff
@@ -0,0 +1,296 @@
+/**
```

Comment:
This file shouldn’t be added here. There must be something wrong with the install or the file that needs this include. Did you define `ATSAM4S` at the top?

Reply:
When building the app for fvm, the macros for `PIO_P*` are referenced. This is here to provide those macros for the POSIX platform headers. Should I define `ATSAM4S` in posix.h?

Reply:
A build for posix should still have access to the SAM headers, as they are installed. I can build the app for FVM fine on macOS so I think we have another case of this doesn't d.t.r.t on Linux.

Reply:
Check `build/include` to see what's copied to `$PREFIX/include`. IIRC `PIO.h` is under `flipper/atsam4s/asf` somewhere.

Reply:
How can both the posix and atsam4s headers be included? The carbon.h file included in flipper.h will include the atsam4s headers if `ATSAM4S` is defined, but then it won't include the posix headers. Based on [lines 7-15 of flipper/carbon.h](https://github.com/flipper-io/flipper/blob/eafc253ae26d79f78379dd85629c2bf67ab000a9/carbon/include/flipper/carbon.h#L7-L15).

Reply:
I just pushed 11fc8c654c8a2443225ed58635820b6e6242aedf, which gets the app to compile, but still not link, for me. Does that work on your end?